integration-cli/docker_cli_events_test.go

```diff
@@ -66,7 +66,7 @@ func (s *DockerSuite) TestEventsContainerFailStartDie(c *check.C) {
 
 func (s *DockerSuite) TestEventsLimit(c *check.C) {
 	for i := 0; i < 30; i++ {
-		dockerCmd(c, "run", "busybox", "echo", strconv.Itoa(i))
+		go dockerCmd(c, "run", "--rm", "busybox", "true")
```

Comment:
I'm afraid there's a race there: we're starting 30 goroutines but immediately call for `docker events`, I don't think we have any guarantee to reliability get the 64 expected events in the end.

I'd add a `sync.WaitGroup`: @LK4D4 what do you think?